packages/charts/src/utils.test.ts

```diff
@@ -7,6 +7,7 @@ import {
   normalizeToPercent,
   calculateRange,
   EDGE_TICK_ALIGN,
+  calculateColorScale,
```

Comment:
After my refactoring there is a separate directory for `utils`. We aiming right now to avoid single utils file that address multiple domains.  
Please, could You move the function to separate file in utils directory - for example. `utils/colors.utils.ts`;